service/handlers_test.go

```diff
@@ -382,44 +385,21 @@ func TestSignHandlerErrorKeyStore(t *testing.T) {
 
 func TestVersionHandler(t *testing.T) {
 
-	config := ConfigSettings{Version: "1.2.5"}
-	Environ = &Env{Config: config}
+	config := config.Settings{Version: "1.2.5"}
+	datastore.Environ = &datastore.Env{Config: config}
 
 	result, _ := sendRequestVersion(t, "GET", "/v1/version", nil)
 
-	if result.Version != Environ.Config.Version {
-		t.Errorf("Incorrect version returned. Expected '%s' got: %v", Environ.Config.Version, result.Version)
-	}
-
-}
-
-func TestVersionHandlerNilEnviron(t *testing.T) {
```

Comment:
This was testing a local env variable and the package Environ variable being set when it was nil. The local env variable has been removed and Environ is always initialized, so it doesn't make sense any more.